tidb-cloud/tidb-cloud-import-local-files.md

```diff
@@ -90,7 +90,7 @@ To import only some specified columns, you can use the MySQL client to connect y
 
 ```sql
 CREATE TABLE `import_test` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `id` int NOT NULL AUTO_INCREMENT,
```

Comment:
`CREATE TABLE` is OK, but for output I assume the documentation will be synced with different repo/version?

Reply:
@qiancai while this `CREATE TABLE` statement would work on both current and future versions of TiDB Cloud Serverless, maybe we should exclude `tidb-cloud/` for now?

@mjonss cloud uses a different TOC: `TOC-tidb-cloud.md` but the same repo. Not sure what branch is used now

Reply:
@dveeden Yes, I agree with you. As the latest TiDB Cloud specific content is still maintained in [release-7.5 -> tidb-cloud](https://github.com/pingcap/docs/tree/release-7.5/tidb-cloud) folder, we can skip this change in the master -> tidb-cloud folder.  When v8.5 becomes the default version of TiDB Dedicated, we can update the content in the tidb-cloud folder accordingly.